pulseapi/profiles/urls.py

```diff
@@ -1,11 +1,17 @@
 from django.conf.urls import url
 
 from pulseapi.profiles.views import (
+    UserProfileAPIView,
     UserProfilePublicAPIView,
     UserProfilePublicSelfAPIView,
 )
 
 urlpatterns = [
+    url(
```

Comment:
I would suggest that we not create a new route just for update. We have the `/myprofile` route to update the profile so let's stick to using that. Maybe we can add a comment here to explain that create/update/delete are only allowed on your own profile and therefore we have a separate route (without the pk) declared in the main urls.py file.